content/blog/2019-02-01-software-peer-review-news.md

```diff
@@ -0,0 +1,103 @@
+date: 2019-02-01
+authors:
+  - Maëlle Salmon
+  - Brooke Anderson
+  - Scott Chamberlain
+  - Anna Krystalli
+  - Lincoln Mullen
+  - Karthik Ram
+  - Noam Ross
+  - Melina Vidoni
+categories: blog
+topicid: 
+tags:
+  - Software Peer Review
+---
+
+rOpenSci's [suite of packages](/packages/) is comprised of contributions from staff engineers and the wider R community, bringing considerable diversity of skills, expertise and experience to bear on the suite. How do we ensure that every package is held to a high standard? That's where our software review system comes into play: packages contributed by the community undergo a **transparent, constructive, non adversarial and open review process**. For that process relying mostly on **volunteer work**, *[associate editors](https://ropensci.github.io/dev_guide/softwarereviewintro.html#associateditors)* manage the incoming flow and ensure progress of submissions; _authors_ create, submit and improve their package; *[reviewers](https://ropensci.github.io/dev_guide/#reviewers)*, two per submission, examine the software code and user experience.
+
+We are quite proud of our Software Peer Review system and do our best improving it over time. In this post, we summarize some of the more important  recent changes to Software Peer Review at rOpenSci. The most recent information can always be found at [ropensci.org/software-review/](https://ropensci.org/software-review/).
+
+### A new and clearer name
+
+Up until December, our system was called _rOpenSci onboarding_ which was short but potentially misleading given the general understanding of "onboarding" as adding new employees or volunteers to an organization. We switched to _rOpenSci Software Peer Review_ which shows the system draws elements from both _Peer Review_ as known in academia, and _Software Review_ as known in software development. The discussion about the name change started [in July 2017](https://github.com/ropensci/software-review-meta/issues/11); we're glad to have finally done it!
+
+### Four new editors
+
+{{< figure src="/img/blog-images/2019-02-01-software-review-news/team.png" caption="Editorial team. From left to right and top to bottom: Brooke Anderson, Scott Chamberlain, Anna Krystalli, Lincoln Mullen, Karthik Ram, Noam Ross, Maëlle Salmon, Melina Vidoni." alt="Editorial team. From left to right and top to bottom: Brooke Anderson, Scott Chamberlain, Anna Krystalli, Lincoln Mullen, Karthik Ram, Noam Ross, Maëlle Salmon, Melina Vidoni." >}}
+
+Since our [last post about updates in the system in September 2017](/blog/2017/09/11/software-review-update/), the editorial team has doubled in size to handle all incoming submissions whilst not overburdening editors. We are delighted to have welcomed Anna Krystalli, Lincoln Mullen, Brooke Anderson and Melina Vidoni. Thanks to them for joining us! Read more [about Anna and Lincoln](/blog/2018/06/22/new_editors/) and [about Brooke and Melina] (blog/2019/01/31/more_editors/).
```

Comment:
```suggestion
Since our [last post about updates in the system in September 2017](/blog/2017/09/11/software-review-update/), the editorial team has doubled in size to handle all incoming submissions whilst not overburdening editors. We are delighted to have welcomed Anna Krystalli, Lincoln Mullen, Brooke Anderson and Melina Vidoni. Thanks to them for joining us! Read more [about Anna and Lincoln](/blog/2018/06/22/new_editors/) and [about Brooke and Melina] (/blog/2019/01/31/more_editors/).
```
fix broken link that I caused